ui/pages/settings/advanced-tab/advanced-tab.stories.js

```diff
@@ -1,48 +1,58 @@
+    dismissSeedBackUpReminder: { control: 'boolean' },
+    setAutoLockTimeLimit: { action: 'setAutoLockTimeLimit' },
+    setShowFiatConversionOnTestnetsPreference: {
+      action: 'setShowFiatConversionOnTestnetsPreference',
+    },
+    setShowTestNetworks: { action: 'setShowTestNetworks' },
+    setThreeBoxSyncingPermission: { action: 'setThreeBoxSyncingPermission' },
+    setIpfsGateway: { action: 'setIpfsGateway' },
+    setLedgerTransportPreference: { action: 'setLedgerTransportPreference' },
+    setDismissSeedBackUpReminder: { action: 'setDismissSeedBackUpReminder' },
+    setUseNonceField: { action: 'setUseNonceField' },
+    setHexDataFeatureFlag: { action: 'setHexDataFeatureFlag' },
+    displayWarning: { action: 'displayWarning' },
+    history: { action: 'history' },
+    showResetAccountConfirmationModal: {
+      action: 'showResetAccountConfirmationModal',
+    },
+    setAdvancedInlineGasFeatureFlag: {
+      action: 'setAdvancedInlineGasFeatureFlag',
+    },
+  },
 };
 
-export const DefaultStory = () => {
+export const DefaultStory = (args) => {
   return (
     <div style={{ flex: 1, height: 500 }}>
-      <AdvancedTab
-        setAutoLockTimeLimit={() => undefined}
-        setShowFiatConversionOnTestnetsPreference={() => undefined}
-        setShowTestNetworks={() => undefined}
-        setThreeBoxSyncingPermission={() => undefined}
-        setIpfsGateway={() => undefined}
-        setLedgerTransportPreference={() => undefined}
-        setDismissSeedBackUpReminder={() => undefined}
-        setUseNonceField={() => undefined}
-        setHexDataFeatureFlag={() => undefined}
-        displayWarning={() => undefined}
-        history={{ push: () => undefined }}
-        showResetAccountConfirmationModal={() => undefined}
-        setAdvancedInlineGasFeatureFlag={() => undefined}
-        warning={text('Warning', 'Warning Sample')}
-        ipfsGateway="ipfs-gateway"
-        useNonceField={boolean('Customize Transaction Nonce', false)}
-        sendHexData={boolean('Show Hex Data', false)}
-        advancedInlineGas={boolean('Advanced Inline Gas', false)}
-        showFiatInTestnets={boolean('Show Conversion on Testnets', false)}
-        threeBoxSyncingAllowed={boolean(
-          'Sync data with 3Box (experimental)',
-          false,
-        )}
-        threeBoxDisabled={boolean('3Box Disabled', false)}
-        useLedgerLive={boolean('Use Ledger Live', false)}
-        dismissSeedBackUpReminder={boolean(
-          'Dismiss recovery phrase backup reminder',
-          false,
-        )}
-      />
+      <AdvancedTab ipfsGateway="ipfs-gateway" {...args} />
     </div>
   );
 };
 
 DefaultStory.storyName = 'Default';
+DefaultStory.args = {
+  warning: 'Warning Sample',
+  useNonceField: false,
+  sendHexData: false,
+  advancedInlineGas: false,
+  showFiatInTestnets: false,
+  threeBoxSyncingAllowed: false,
+  threeBoxDisabled: false,
+  useLedgerLive: false,
+  dismissSeedBackUpReminder: false,
+};
```

Comment:
Nice work you could improve the intuitiveness of this story by using `useArgs` to make the toggles work in the story as well. You can see how it's used in the [snap-settings-card](https://metamask.github.io/metamask-storybook/index.html?path=/story/ui-components-app-flask-snap-settings-card-snap-settings-card-stories-js--default-story&args=status:running;isEnabled:false&knob-Advanced%20Inline%20Gas=true&knob-Asset=NATIVE&knob-Customize%20Transaction%20Nonce=true&knob-Show%20Hex%20Data=true&knob-Stage=ADD_RECIPIENT&knob-Warning=Something%20else) `ui/components/app/flask/snap-settings-card/snap-settings-card.js` 


https://user-images.githubusercontent.com/8112138/149008996-bc7a75e4-8ae3-4efc-af66-4c8d409247a3.mov

Reply:
@georgewrmarshall I added `useArgs` to make the toggles work in the story as well.
You can see changes here: 3123dbb12e2dd7ae6aea5e4bc5b0b6e4f23aab41
Is it okay now?

Reply:
Nicely done!